source/operations/concepts.rst

```diff
@@ -107,6 +107,7 @@ MinIO :ref:`distributed deployments <minio-mnmd>` support expansion and decommis
 
 Expansion consists of adding one or more :ref:`server pools <minio-intro-server-pool>` to an existing deployment.
 Each server pool consists of dedicated nodes and storage that contribute to the overall capacity of the deployment.
+The size of an individual server pool is fixed at the time it is created.
```

Comment:
Just trying to get it out of passive voice.

```suggestion
Once set, a server pool's size cannot change.
```

Reply:
I'm very confused at what GitHub just did, your suggestion is referencing the previous version of the text.

Reply:
Odd, very odd